Client/qt/dialog.ui

```diff
@@ -0,0 +1,241 @@
+        </property>
+        <property name="maximum">
+          <number>19</number>
+        </property>
+        <property name="orientation">
+          <enum>Qt::Horizontal</enum>
+        </property>
+        <property name="tickPosition">
+          <enum>QSlider::TicksAbove</enum>
+        </property>
+      </widget>
+      <widget class="QLabel" name="label_2">
+        <property name="geometry">
+          <rect>
+            <x>290</x>
+            <y>80</y>
+            <width>131</width>
+            <height>17</height>
+          </rect>
+        </property>
+        <property name="text">
+          <string>Ambient Sound Level</string>
+        </property>
+      </widget>
+      <widget class="QCheckBox" name="focusOnVoiceCheckBox">
+        <property name="enabled">
+          <bool>false</bool>
+        </property>
+        <property name="geometry">
+          <rect>
+            <x>10</x>
+            <y>110</y>
+            <width>111</width>
+            <height>23</height>
+          </rect>
+        </property>
+        <property name="text">
+          <string>Focus on Voice</string>
+        </property>
+        <property name="checkable">
+          <bool>true</bool>
+        </property>
+      </widget>
+      <widget class="QCheckBox" name="ambientSoundControlCheckBox">
+        <property name="geometry">
+          <rect>
+            <x>10</x>
+            <y>30</y>
+            <width>261</width>
+            <height>23</height>
+          </rect>
+        </property>
+        <property name="text">
+          <string>Ambient Sound Control</string>
+        </property>
+        <property name="checkable">
+          <bool>true</bool>
+        </property>
+        <property name="checked">
+          <bool>true</bool>
+        </property>
+      </widget>
+    </widget>
+    <widget class="QGroupBox" name="deviceDiscoveryGroupBox">
+      <property name="geometry">
+        <rect>
+          <x>10</x>
+          <y>0</y>
+          <width>431</width>
+          <height>221</height>
+        </rect>
+      </property>
+      <property name="title">
+        <string>Device Discovery</string>
+      </property>
+      <widget class="QPushButton" name="connectButton">
+        <property name="enabled">
+          <bool>false</bool>
+        </property>
+        <property name="geometry">
+          <rect>
+            <x>10</x>
+            <y>190</y>
+            <width>101</width>
+            <height>25</height>
+          </rect>
+        </property>
+        <property name="text">
+          <string>C&amp;onnect</string>
+        </property>
+      </widget>
+      <widget class="QPushButton" name="refreshButton">
+        <property name="geometry">
+          <rect>
+            <x>120</x>
+            <y>190</y>
+            <width>131</width>
+            <height>25</height>
+          </rect>
+        </property>
+        <property name="text">
+          <string>&amp;Refresh devices</string>
```

Comment:
Having shortcuts so easily is really cool!

Reply:
Indeed. I always try to make them for buttons at least in a dialog app like this. This syntax is the same as on [Windows](https://docs.microsoft.com/en-us/windows/win32/menurc/about-menus#menu-access-keys).

Reply:
Neat, TIL